sale_order_packaging_info/views/res_config_settings_views.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo>
+    <record id="res_config_settings_view_form" model="ir.ui.view">
+        <field
+            name="name"
+        >res.config.settings.view.form.inherit.sale_order_packaging_info</field>
```

Comment:
why `inherit` in the name? is that usual?

Reply:
I probably copied this from some other module that inherits res_config_settings_view_form and slightly edited it. Corrected, but the name field doesn't really matter.